docs/demo/components/Button/button-basic.demo.tsx

```diff
@@ -0,0 +1,13 @@
+import LiveDemoEditor from '@/DemoLiveEditor';
+import { Button } from '@marigold/components';
+
+export const BasicButton = () => {
+  return (
+    <LiveDemoEditor
+      scope={{ Button }}
+      code={`
+            <button className='text-white rounded-lg'>demo button</button>
```

Comment:
Can this work with TypeScript code too? Because this is what we want to display :)

Reply:
it works fine for reading typescript and i tried  an example for this, but you can't see what types are on hover :"
is that fine ?!

Reply:
you can run the app and go to `http://localhost:3000/counter`
And you can view the file for this on `docs/demo/components/Counter`